actionlib_tutorials/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ generate_messages(DEPENDENCIES std_msgs actionlib_msgs)
 
 ## Define catkin exports
 catkin_package(
-  CATKIN_DEPENDS roscpp actionlib
```

Comment:
There are programs in this package that directly `#include <ros/ros.h>` so it is better to express directly the dependency on `roscpp`.